eventonica-api/.gitignore

```diff
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

Comment:
I think it's fine to just add this to the toplevel `.gitignore` rather than having one for each subproject.